en/components/combo_cascading.md

```diff
@@ -1,90 +1,66 @@
+The [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and [igx-select]({environment:angularApiUrl}/classes/igxselectcomponent.html) exposes events, properties and methods that allows to easily construct cascading scenario, either using one of the controls or both of them.
 </p>
 <div class="divider"></div>
 
 ### Demo
 
-The following sample demonstrates scenario, using two [igx-drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) and [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) components:
+The following sample demonstrates scenario, using two [igx-select]({environment:angularApiUrl}/classes/igxselectcomponent.html) and [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) components:
 
 <div class="sample-container loading" style="height: 540px;">
     <iframe id="cascading-combos-sample" frameborder="0" seamless width="100%" height="100%" src="{environment:demosBaseUrl}/lists/cascading-combos" onload="onSampleIframeContentLoaded(this);"></iframe>
 </div>
 <div>
-    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="cascading-combos-sample" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="cascading-combos-sample" data-demos-base-url="{environment:demosBaseUrl}">StackBlitz 에서보기</button>
 </div>
 <div class="divider--half"></div>
 
 ### Usage
-To get started import the `IgxComboModule`, `IgxDropDownModule`, `IgxIconModule`, `IgxInputGroupModule` in the **app.module.ts** file:
+To get started import the `IgxComboModule`, `IgxSelectModule` in the **app.module.ts** file:
 
 ```typescript
 // app.module.ts
 
 ...
-import { IgxComboModule, IgxDropDownModule, IgxIconModule, IgxInputGroupModule } from 'igniteui-angular';
+import { IgxComboModule, IgxSelectModule } from 'igniteui-angular';
 
 @NgModule({
     ...
-    imports: [..., IgxComboModule, IgxDropDownModule, IgxIconModule, IgxInputGroupModule],
+    imports: [..., IgxComboModule, IgxSelectModule],
     ...
 })
 export class AppModule {}
 ```
 
-In the sample below are used an [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and two [igx-drop-down]({environment:angularApiUrl}/classes/igxdropdowncomponent.html) components along with corresponding input elements. The API of the both components are used to get selected item from one and load data source for the next either drop down or combo, clear selection or reset data source.
+In the sample below are used an [igx-combo]({environment:angularApiUrl}/classes/igxcombocomponent.html) and two [igx-select]({environment:angularApiUrl}/classes/igxselectcomponent.html) components. The API of the both components are used to get selected item from one and load data source for the next either select or combo, clear selection or reset data source.
```

Comment:
The wording on `In the sample below are used` seems off to me. I think that something along the lines of `The sample below uses` could work.